index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <header class="Header">
+      <nav>
+        <h1>Elie Harfouche</h1>
+        <div class="lines">
+          <div class="line1"></div>
+          <div class="line2"></div>
+          <div class="line3"></div>
+        </div>
+      </nav>
```

Comment:
- [ ] Good job so far but I think you need to change something at this section, your `<h1>` is a logo and the logo is supposed to be in the `<a>` tag, kindly fix it :+1: :+1:.
- [ ] For the hamburger button you suppose to use the image not the class that you style, and also it suppose to be in the `<nav>` tag not the logo :upside_down_face: :upside_down_face:
Example of how you can implement it:
```
  <a class="logo" href="#">Elie Harfouche</a>
  <nav class="nav-bar">
     <a href="#">
         <img src="image-link">
     </a>
  </nav>
```